lib/y2logsstep.pm

```diff
@@ -99,6 +99,12 @@ sub check_and_record_dependency_problems {
             send_key 'alt-a';
         };
         send_key 'alt-o';
+        if (check_var('VERSION', '15')) { # SLE 15 has unsupport packages, workaround them - rbrown 04/07/2017
+            while (check_screen('sle-15-unsupported-packages')) {
+                record_soft_failure 'bsc#1047337';
```

Comment:
Cannot we stuck in infinite loop if for whatever reason screen remains same? Even though it's good enough as a workaround.